### PR TITLE
Add table layer API

### PR DESCRIPTION
**Pull request by homm**

### CODE REVIEW

**Comment by homm** [comment]
[Drop dynamo mapper in robots](https://trello.com/c/186sPpbb/415-drop-dynamo-mapper-in-robots)

**Comment by andrewshkovskii** [comment]
Should we review this one or not?

File: asyncdynamo/tables.py [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+            range_key_conditions=range_key_conditions,
+            object_hook=object_hook, **kwargs)
+
+        if kwargs['error']:
+            raise kwargs['error']
+
+        last_key = None
+        if 'LastEvaluatedKey' in resp:
+            last_key = resp['LastEvaluatedKey']
+            last_key = last_key['HashKeyElement'], last_key['RangeKeyElement']
```
Comment (andrewshkovskii) [inline asyncdynamo/tables.py:93]: `last_key` is a tuple - is that correct?

**Review by andrewshkovskii — CHANGES_REQUESTED** [review]

File: asyncdynamo/tables.py (hunk 1 above)
Comment (homm) [reply asyncdynamo/tables.py:93]: Right. Dynamo tables have up to two key fields. If only one is exist, None is correct value for the range key

**Review by andrewshkovskii — APPROVED** [review]

**Pull request merged**